build.gradle.kts

```diff
@@ -86,6 +90,30 @@ yamlValidator {
     isSearchRecursive = true
 }
 
+sonarqube {
+    // If you want to use this logcally a sonarLogin has to be provide, either via Username and Password
+    // or via token, https://docs.sonarqube.org/latest/analysis/analysis-parameters/
+    properties {
+        // Default properties if somebody wants to execute it locally
+        property("sonar.projectKey", "aepfli_junit-pioneer") // needs to be changed
+        property("sonar.organization", "aepfli") // needs to be changed
+        property("sonar.host.url", "https://sonarcloud.io")
+        /*
+        Travis CI provides a Environment Variable called $SONARQUBE_SCANNER_PARAMS, if it is provided and
+        it is populated it will be a json string, if not, it will print the name.
+         */
+
+        val travisSonarProps: String? = System.getenv("SONARQUBE_SCANNER_PARAMS")
```

Comment:
normally there should be no logic within a build script. and i agree here, i just thought this might be a little overhead for now, to extract this into an own gradle file - but if requested i can offer to do this, or provide this on a later stage.

Reply:
For me it's okay. An own file for this one liner would be overkill in my opinion.